src/pages/[platform]/build-a-backend/storage/download/index.mdx

```diff
@@ -129,12 +129,35 @@ await getUrl({ key: 'filename.txt', options: { expiresIn: 60 } });
 
 Download a file to in-memory buffer.
 
+<BlockSwitcher>
+<Block name="Path">
 ```javascript
 import { downloadData } from 'aws-amplify/storage';
 
 // Downloads file content to memory
 const { body, eTag } = await downloadData({
-  key,
+  path: filename,
+  data: file,
+  options: {
+    onProgress: (event) => {
+      console.log(event.transferredBytes);
+    } // optional progress callback
+    bytesRange: {
+      start: 1024,
+      end: 2048
+    } // optional bytes range parameter to download a part of the file, the 2nd MB of the file in this example
+  }
+}).result;
+```
+</Block>
+<Block name="Key (Deprecated)">
```

Comment:
This will not be deprecated right away or has not yet been deprecated at the moment of publishing this new documentation right? 😅 `(going to be deprecated)?`

Reply:
Yes. I added this to comment above. Makes sense to make it clear